bin/ch/Helpers.cpp

```diff
@@ -7,30 +7,41 @@
 #include <sys/stat.h>
 
 //TODO: x-plat definitions
-#ifdef _WIN32
 #define MAX_URI_LENGTH 512
+#ifdef _WIN32
+#define TTD_MAX_FILE_LENGTH MAX_PATH
 #define TTD_HOST_PATH_SEP "\\"
+#else
+#define TTD_MAX_FILE_LENGTH MAX_URI_LENGTH
```

Comment:
`MAX_URI_LENGTH` definition looks redundant.

Reply:
`MAX_URI_LENGTH` is needed in functions `Helpers::CreateTTDDirectoryAsNeeded` (L393) and `Helpers::TTCreateStreamCallback` (L457).

Reply:
Or do you mean change this line to `#define TTD_MAX_FILE_LENGTH 512`?

Reply:
Yes.